admin/replication/LoadReplicationChanges

```diff
@@ -175,6 +183,12 @@ if (
         $sql->do('TRUNCATE dbmirror_pending CASCADE');
         $sql->auto_commit(1);
         $sql->do('TRUNCATE dbmirror_pendingdata CASCADE');
+        $sql->auto_commit(1);
+        $sql->do('TRUNCATE dbmirror2.pending_data CASCADE');
+        $sql->auto_commit(1);
+        $sql->do('TRUNCATE dbmirror2.pending_keys CASCADE');
+        $sql->auto_commit(1);
+        $sql->do('TRUNCATE dbmirror2.pending_ts CASCADE');
```

Comment:
After these tables are cleared, the broken packet will be re-downloaded ("in case the packet was fixed", per the comment above) and applied again. Since the previously-aborted load will have applied any changes up until the first broken one, we also need a way to skip insertions that conflict with existing primary keys. The `--ignore-conflicts` flag is *only* passed if both `$aborted_sequence` and `$known_broken_sequences{$aborted_sequence}` are defined.